mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/CassandraMailboxManagerTest.java

```diff
@@ -37,37 +38,36 @@
 import org.apache.james.mailbox.cassandra.modules.CassandraSubscriptionModule;
 import org.apache.james.mailbox.cassandra.modules.CassandraUidModule;
 import org.junit.After;
-import org.junit.AfterClass;
-import org.junit.BeforeClass;
+import org.junit.Before;
+import org.junit.ClassRule;
 
 public class CassandraMailboxManagerTest extends MailboxManagerTest {
 
-    private static CassandraCluster cassandra;
+    @ClassRule public static DockerCassandraRule cassandraServer = new DockerCassandraRule();
     
-    @BeforeClass
-    public static void init() {
-        cassandra = CassandraCluster.create(
-                new CassandraModuleComposite(
-                    new CassandraAclModule(),
-                    new CassandraMailboxModule(),
-                    new CassandraMessageModule(),
-                    new CassandraBlobModule(),
-                    new CassandraMailboxCounterModule(),
-                    new CassandraMailboxRecentsModule(),
-                    new CassandraFirstUnseenModule(),
-                    new CassandraUidModule(),
-                    new CassandraModSeqModule(),
-                    new CassandraSubscriptionModule(),
-                    new CassandraAttachmentModule(),
-                    new CassandraDeletedMessageModule(),
-                    new CassandraAnnotationModule(),
-                    new CassandraApplicableFlagsModule()));
-    }
-
-    @AfterClass
-    public static void close() {
-        cassandra.close();
+    private CassandraCluster cassandra;
+    
+    @Before
+    public void setup() throws Exception {
+        CassandraModuleComposite modules = new CassandraModuleComposite(
```

Comment:
I don't agree with you, each tests should be responsible of what is loaded.

Reply:
Sure, I agree about the responsibility. I am not arguing on that.

What I propose is:
- create a MailboxModules interface
- have this constant defined in it
- replace it when strictly equals

Reply:
I don't like it, it's over-reuse for me, it's similar but the lifecycle is not the same : once you reference a common constant, it will leave independently from tests and that's not a good idea.